imports/ui/pages/profile.jsx

```diff
@@ -51,9 +52,34 @@ class _Profile extends Component {
         name: fileURL})
       let image = {UID: UID}
       const existing = Images.findOne(image)
-
+      const settings = Settings.findOne()
+      
       if (existing) {
-        this.saveProfileImage(existing.url)
+        console.log(existing.url)
```

Comment:
I don't think you need to do anything to update the links if the files already exist; The getters for the image and thumbnail will already have run and made the updates when the profile.jsx component and page_container mount?